infra/image/system-service/fixipaip.sh

```diff
@@ -1,26 +1,82 @@
 #!/bin/bash -eu
 
+function validFQDN()
```

Comment:
really minor and cosmetic: shell naming conventions lowercase and separate words with underscore, what about 

valid_fqdn()

Reply:
Fixed